lib/signaling/v2/localparticipant.js

```diff
@@ -53,6 +75,21 @@ class LocalParticipantV2 extends LocalParticipantSignaling {
     });
   }
 
+  /**
+   * Update the {@link BandwidthProfileOptions}.
+   * @param {BandwidthProfileOptions} bandwidthProfile
+   */
+  setBandwidthProfile(bandwidthProfile) {
+    if (!isDeepEqual(this._bandwidthProfile, bandwidthProfile)) {
+      // NOTE(mmalavalli): Object.assign() copies the values of only
```

Comment:
no need to mention `Object.assign` as you are not using it here :)

Reply:
@makarandp0 I think it's good to specify why we can't use `Object.assign()` here.